oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/storage/model/StorageModels.java

```diff
@@ -99,14 +109,18 @@ private void retrieval(Class<?> clazz,
                 Column column = field.getAnnotation(Column.class);
                 modelColumns.add(
                     new ModelColumn(
-                        new ColumnName(modelName, column.columnName()), field.getType(), column.matchQuery(), column
-                        .storageOnly(), column.dataType().isValue(), column.length()));
+                        new ColumnName(modelName, column.columnName()), field.getType(), column.matchQuery(),
+                        column
+                            .storageOnly(), column.dataType().isValue(), column.length()
+                    ));
```

Comment:
```suggestion
                        new ColumnName(modelName, column.columnName()), field.getType(), column.matchQuery(),
                        column.storageOnly(), column.dataType().isValue(), column.length()
                    ));
```

These lines seems should be layout better.

Reply:
already change, Since code style is not installed this time, there will not be so many formatting problems next time